test/controllers/DeclarationControllerSpec.scala

```diff
@@ -0,0 +1,83 @@
+package controllers
+
+import play.api.data.Form
+import play.api.libs.json.JsString
+import uk.gov.hmrc.http.cache.client.CacheMap
+import navigation.FakeNavigator
+import connectors.FakeDataCacheConnector
+import controllers.actions._
+import play.api.test.Helpers._
+import forms.DeclarationFormProvider
+import models.NormalMode
+import pages.DeclarationPage
+import play.api.mvc.Call
+import views.html.declaration
+
+class DeclarationControllerSpec extends ControllerSpecBase {
+
+  def onwardRoute = Call("GET", "/foo")
+
+  val formProvider = new DeclarationFormProvider()
```

Comment:
@russellthorn 
My guess is that you forgot to push the file `DeclarationFormProvider` in the package `forms` 

This is not compiling

Reply:
I think I can fix this for @russellthorn 
But please review my changes to double check

Reply:
I have done it